dojo/tools/anchore_engine/parser.py

```diff
@@ -1,59 +1,45 @@
-__author__ = 'jaguasch'
-
 import json
-from datetime import datetime
 
 from dojo.models import Finding
 
 
 class AnchoreEngineScanParser(object):
     def get_findings(self, filename, test):
-        tree = filename.read()
-        try:
-            data = json.loads(str(tree, 'utf-8'))
-        except:
-            data = json.loads(tree)
+        data = json.load(filename)
         dupes = dict()
-        find_date = datetime.now()
-
         for item in data['vulnerabilities']:
-            categories = ''
-            language = ''
-            mitigation = ''
-            impact = ''
-            references = ''
-            findingdetail = ''
-            title = ''
-            group = ''
-            status = ''
-            cve = None
+            cve = item.get('vuln')
 
             title = item['vuln'] + ' - ' + item['package'] + '(' + item['package_type'] + ')'
```

Comment:
This could then also read `cve` and follow the changes you made to use `item.get()`?

Reply:
I don't want to rewrite entirely the parser.
I made a lot of fixes and code cleaning with this PR, I think this version is better.